src/aac/__init__.py

```diff
@@ -1,11 +1,14 @@
+from os.path import abspath
+
 from pluggy import HookimplMarker, HookspecMarker
 
 PLUGIN_PROJECT_NAME = "aac"
+PROJECT_ROOT_DIR = abspath("../../")
```

Comment:
I don't see where this is used or necessary. What's the purpose behind this?

Reply:
It's not used anymore. I was originally using it with my version of the `FileSystemLoader` for the templates - I'll remove the references.

Reply:
```suggestion
```